### DISCUSSION TITLE
Unable install isce2 on macOS Monterey (v12.6.1)

### DISCUSSION THREAD

**Discussion by rprasanth49** [isce-framework/isce2#610] [Q&A]
Hello All,

I am new to Mac world and isce. I am trying to get isce2 installed on mac (Monterey 12.6.1) but facing various issue while trying to build GCC in the process. I would not move past building gcc on every method I tried to install isce2. I have listed all the methods I tried so far below. Is there something I am missing on the mac or in my process of doing the installation? Is there any documentation to install the isce with the latest libraries on the latest version of mac? I cannot seem to find anything that is working. Sorry for asking this type of question here but this is my second week trying to install isce2 and still no success. Looking for some kind of help on setting up the system.

Methods tried so far to install the isce2:

1. Tried to install isce2 and all dependencies individually using Anaconda but I was always getting isce2 is not installed from Python.
2. Installed MacPorts and tried to install dependencies and it failed the first to install the gcc7 saying unsupported mac OS version.
3. Tried to use the .install.sh file to install the dependencies and isce but again building GCC failed to stating that \lib\cpp fails sanity check. Used this link has reference - http://etdedata.gein.noa.gr/ISCE_UNAVCO/ISCE-2.0.0_201511.pdf
4. Formatted the mac and started the fresh install using install.sh from the setup folder in isce2 but faced the same issue as in method 3 (\lib\cpp fails sanity check)
5. I even tried Google Colab by following this link (https://colab.research.google.com/github/SERVIR/ForestStandHeight/blob/master/Section_2/Install_ISCE.ipynb) and still no luck.

Thanks,
Prasanth

**Comment by fdelgadodelapuente** [comment]
Hi Prasanth,

Follow the instructions here
https://www.overleaf.com/project/5babb2b0a601937d5d27c762

Tested with Monterey and macports.  If you use conda you will have to update them.

Francisco

**Reply by fdelgadodelapuente** [reply]
Version 12.3.1 is Monterey

![Screen Shot 2022-11-01 at 18 59 52](https://user-images.githubusercontent.com/23100180/199350053-b77fd35e-67e5-4f2a-9c29-cf7787129656.png)

**Reply by rprasanth49** [reply]
Thank you!
